README.rst

```diff
@@ -52,7 +49,12 @@ Set up the environment and compile the package using the following commands:
     make compile
     make install-dev
 
-And finnally run the tests:
+.. note::
+    On Mac OS X, you might need to install openssl `brew install openssl` and then
+    export the location of the libs: `export LDFLAGS="-L/usr/local/opt/openssl/lib"`
+    and `export CPPFLAGS="-I/usr/local/opt/openssl/include"`.
```

Comment:
should this be added here [1] together with the other comment?

[1] https://github.com/pfreixes/acsylla/pull/22/files#diff-88b99bb28683bd5b7e3a204826ead112R36